### PR TITLE
Add a Shoes app to find and remove duplicate examples under the examples dir

### PR DESCRIPTION
**Pull request by noahgibbs**
I also removed a bunch of examples manually before deciding "hey, this would be a fantastic little Shoes app."

I made a minor change to Scarpe::ListItem#change (returning self) so we can chain calls to it, and then used that in the app here.

The app works. You have to hit "refresh" before doing anything. But then select an item from the list and hit "There Can Be Only One" and it'll git-rm all but the first copy and update the list. So, uh, only hit that button more than once if you mean it.

But on the plus side, it's a git-rm, so it's not gone forever.

I didn't delete all the duplicate examples so other people can try it out if they want :-)

Screenshot 2023-06-15 at 10 41 33

Screenshot 2023-06-15 at 10 41 47

### CODE REVIEW

**Comment by Schwad** [comment]
o.m.g.

File: examples/find_dup_examples.rb [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+def highlander_the_selected_item
+  selected = $dup_list.selected_item
+  if !selected || selected == "" || !selected.include?(DIVIDER)
+    $bottom_display.replace("Select an item first!")
+    return
+  end
+
+  files = selected.split(DIVIDER)
+  first_content = File.read files[0]
+  files[1..-1].each do |sean_connery|
```
Comment (Schwad) [inline examples/find_dup_examples.rb:46]: connery

File: examples/find_dup_examples.rb [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    unless $?.success?
+      $bottom_display.replace("Got an error while git-removing #{sean_connery.inspect}: #{$?.inspect}")
+      return
+    end
+  end
+
+  update_dup_list
+end
+
+Shoes.app do
```
Comment (Schwad) [inline examples/find_dup_examples.rb:64]: Do you want to give your amazing app a title? "Deduperizer"?

"DupliVanisher"? "DoubleScrub"? "Kangaroo"? "SeanConnery"? 😂

**Review by Schwad — APPROVED** [review]
Amazing! And this will make me a lot more confident just like, piling in lots of found examples because cleanup will be very doable!

10/10 this should be named, packaged and put in the Shoes store.

File: examples/find_dup_examples.rb (hunk 2 above)
Comment (noahgibbs) [reply examples/find_dup_examples.rb:64]: Given that I'm going with Highlander silliness in the code, maybe I'll call it "Lambert" after Christopher Lambert.

Comment (noahgibbs) [reply examples/find_dup_examples.rb:64]: Dunno, I'll figure something out and add it in the next few minutes.

**Pull request merged**